consensus/spos/bls/blsSubroundsFactory.go

```diff
@@ -142,7 +144,7 @@ func (fct *factory) generateStartRoundSubround() error {
 		return err
 	}
 
-	subroundStartRound, err := NewSubroundStartRound(
```

Comment:
Maybe we can keep subround, as a single word, without caps R everywhere in this file